src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +16,12 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int countTrue = 0;
```

Comment:
it's a bit too complicated, try to solve it using just logic operators &&, ||,  ^ (XOR)

Reply:
done